modules/distribution/src/main/lib/fetch.sh

```diff
@@ -0,0 +1,112 @@
+  fn=$artifact-$version.$ftype
+  path="${group//.//}/$artifact/$version/$fn"
+  download_url=$maven_prefix/$path
+
+  if [ -f $dir/$fn ]; then
+    echo "SUCCESS: Dependency exists - $dir/$fn"
+  else 
+    wget -q $download_url -P $dir
+    if [ $? == 0 ]; then
+      echo "SUCCESS: Dependency downloaded from $download_url"
+    else
+      echo "ERROR: Dependency download failed - $download_url"
+    fi
+  fi
+}
+
+case "$1" in
+ahz)
+  echo "Fetching Hadoop, Accumulo, & Zookeeper dependencies"
+  cd $lib_dir/ahz
+  rm -f *.jar
+  mvn clean package ${*:2}
+  ;;
+extra)
+  echo "Fetching extra Fluo dependencies"
+  download aopalliance:aopalliance:jar:1.0
+  download ch.qos.logback:logback-classic:jar:1.1.3 ./logback
+  download ch.qos.logback:logback-core:jar:1.1.3 ./logback
+  download com.101tec:zkclient:jar:0.3
+  download com.beust:jcommander:jar:1.32
+  download com.google.code.findbugs:jsr305:jar:2.0.1
+  download com.google.code.gson:gson:jar:2.2.4
+  download com.google.guava:guava:jar:13.0.1
+  download com.google.inject:guice:jar:4.0
+  download com.yammer.metrics:metrics-annotation:jar:2.2.0
+  download com.yammer.metrics:metrics-core:jar:2.2.0
+  download commons-collections:commons-collections:jar:3.2.1
+  download commons-configuration:commons-configuration:jar:1.10
+  download commons-io:commons-io:jar:2.4
+  download io.dropwizard.metrics:metrics-core:jar:3.1.1
+  download io.dropwizard.metrics:metrics-graphite:jar:3.1.1
+  download javax.inject:javax.inject:jar:1
+  download log4j:log4j:jar:1.2.17 ./log4j
+  download net.sf.jopt-simple:jopt-simple:jar:3.2
+  download org.apache.curator:curator-client:jar:2.7.1
+  download org.apache.curator:curator-framework:jar:2.7.1
+  download org.apache.curator:curator-recipes:jar:2.7.1
+  download org.apache.kafka:kafka_2.10:jar:0.8.0
+  download org.apache.twill:twill-api:jar:0.6.0-incubating
+  download org.apache.twill:twill-common:jar:0.6.0-incubating
+  download org.apache.twill:twill-core:jar:0.6.0-incubating
+  download org.apache.twill:twill-discovery-api:jar:0.6.0-incubating
+  download org.apache.twill:twill-discovery-core:jar:0.6.0-incubating
+  download org.apache.twill:twill-yarn:jar:0.6.0-incubating
+  download org.apache.twill:twill-zookeeper:jar:0.6.0-incubating
+  download org.hdrhistogram:HdrHistogram:jar:2.1.8
+  download org.mpierce.metrics.reservoir:hdrhistogram-metrics-reservoir:jar:1.1.0
+  download org.ow2.asm:asm-all:jar:5.0.2
+  download org.scala-lang:scala-compiler:jar:2.10.1
+  download org.scala-lang:scala-library:jar:2.10.1
+  download org.scala-lang:scala-reflect:jar:2.10.1
+  download org.slf4j:jcl-over-slf4j:jar:1.7.2
+  download org.slf4j:log4j-over-slf4j:jar:1.7.12 ./logback
+  download org.slf4j:slf4j-api:jar:1.7.12
+  download org.slf4j:slf4j-log4j12:jar:1.7.12 ./log4j
+  download org.xerial.snappy:snappy-java:jar:1.0.5
+  echo "NOTE - The dependencies downloaded have been tested with some versions of Hadoop, Zookeeper, and Accumulo."
```

Comment:
How does this look when you run it?  I am wondering if it needs a an empty line before this note.